deepchecks/checks/integrity/dominant_frequency_change.py

```diff
@@ -42,20 +42,20 @@ def __init__(self, dominance_ratio: float = 2, ratio_change_thres: float = 1.5,
         self.ratio_change_thres = ratio_change_thres
         self.n_top_columns = n_top_columns
 
-    def run(self, dataset, baseline_dataset, model=None) -> CheckResult:
+    def run(self, train_dataset, test_dataset, model=None) -> CheckResult:
```

Comment:
In all places - test_dataset should replace the dataset and train_dataset should replace baseline_dataset, not the other way around.  The order here in the def is fine, but not that later in the code you switched the roles.

Reply:
fixed, please take a look